LoRaEngine/LoraKeysManagerFacade/FacadeStartup.cs

```diff
@@ -42,17 +53,12 @@ public override void Configure(IFunctionsHostBuilder builder)
             var redisCache = redis.GetDatabase();
             var deviceCacheStore = new LoRaDeviceCacheRedisStore(redisCache);
 
-#pragma warning disable CA2000 // Dispose objects before losing scope
-            // Object is handled by DI container.
-            _ = builder.Services.AddSingleton(RegistryManager.CreateFromConnectionString(iotHubConnectionString));
-#pragma warning restore CA2000 // Dispose objects before losing scope
             builder.Services.AddAzureClients(builder =>
-            {
-                _ = builder.AddBlobServiceClient(configHandler.StorageConnectionString)
-                           .WithName(WebJobsStorageClientName);
-            });
+                       {
```

Comment:
Seems that something is odd with the indentation, maybe just related to the GitHub rendering. I guess a ctrl + k + d from Visual Studio will just fix those:

![image](https://user-images.githubusercontent.com/8145578/145402293-6db0caad-9def-4c7c-b896-f3ac5ac9f16c.png)